wireapp/wire-android
# fix: crash when no browser available (WPB-9436)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9436" title="WPB-9436" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-9436</a>  [Android] Playstore crash - Customtab opening on >= Android 11
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

If the app has no browser available to handle opening a URL, the app crashes.

### Solutions

For now, as discussed in https://wearezeta.atlassian.net/browse/WPB-9436?focusedCommentId=124984
We handle it and log it, there is no extra action needed for now.

#### How to Test

Disable and/or delete all browsers in the phone, before the app was crashing, now it just does nothing, but you can see logs for it either locally or in datadog.

### Attachments (Optional)

https://github.com/wireapp/wire-android/assets/5806454/e3d72fdc-298f-423d-8381-cfbba60c6feb


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
